faq/billing.mdx

```diff
@@ -69,6 +69,16 @@ Invoices are issued each month based on the resources' consumption of the previo
 Our reference billing period is the definition of a calendar month in the Coordinated Universal Time (UTC) time zone.
 The automatic payment of invoices occurs in the first few days of a month, once the invoice is issued and is processed by using the preferred payment method of your account.
 
+## What does the status of my invoice mean?
+
+The status of an invoice can be:
+* **Draft**: Concerns the invoice for the current calendar month. The billing period has not yet finished, so the invoice is not finalized and is subject to change.
+* **Stopped**: Concerns invoices where the billing period (calendar month) has ended, but the invoice is not yet issued.
+* **Issued**: Concerns a final invoice awaiting payment. It has a unique number, and is non-repudiable. The invoice charges for the resources consumed over the corresponding billing period.
+* **Paid**: Concerns a fully paid invoice. No further action is required.
+* **Incomplete**: Concerns invoices where the user has provided incorrect billing details or payment information, or elements of this information are missing. The user must update this information as soon as possible to avoid having their account deleted.
```

Comment:
```suggestion
* **Incomplete**: regards invoices where the user has provided incorrect billing details or payment information, or elements of this information are missing. The user must update this information as soon as possible to avoid having their account deleted.
```

Reply:
```suggestion
* **Incomplete**: Refers to invoices where the user has provided incorrect billing details or payment information, or elements of this information are missing. The user must update this information as soon as possible to avoid having their account deleted.
```